### DISCUSSION TITLE
Welcome to Iot-Surveillance-Car Discussions!

### DISCUSSION THREAD

**Discussion by rv2442** [rv2442/Iot-Surveillance-Car#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SawSoeThan** [comment]
I want to know.

why you use two microcontrollers both stm32 and esp8266?

Can you only use eps8266?

**Reply by rv2442** [reply]
Our project had a (college) requirement to utilise stm32, you can interface all sensors directly to esp8266/esp32 if you wish to do so.
